pkg/skaffold/runner/new.go

```diff
@@ -184,7 +184,7 @@ func isImageLocal(runCtx *runcontext.RunContext, imageName string) (bool, error)
 	var pushImages bool
 	if pipeline.Build.LocalBuild.Push == nil {
 		pushImages = cl.PushImages
-		logrus.Debugf("push value not present, defaulting to %t because cluster.PushImages is %t", pushImages, cl.PushImages)
+		logrus.Debugf("push value not present in isImageLocal, defaulting to %t because cluster.PushImages is %t", pushImages, cl.PushImages)
```

Comment:
nit: to clarify this is a function?
```suggestion
		logrus.Debugf("push value not present in isImageLocal(), defaulting to %t because cluster.PushImages is %t", pushImages, cl.PushImages)
```

Reply:
👍